march_hardware_interface/launch/hardware.launch

```diff
@@ -1,6 +1,8 @@
 <launch>
     <arg name="robot" default="march4" doc="The robot to run. Can be: march3, march4, test_joint_linear, test_joint_rotational."/>
 
+    <arg name="reset_imc" default="false" doc="Reset the IMC if this argument is set to true"/>
+
```

Comment:
```suggestion
<launch>
    <arg name="robot" default="march4" doc="The robot to run. Can be: march3, march4, test_joint_linear, test_joint_rotational."/>
    <arg name="reset_imc" default="false" doc="Reset the IMC if this argument is set to true"/>

```
Sorry the comment here was not very clear. I meant remove empty line 3 and empty line 5. That way, the args are grouped, and separated from the rest by a single newline.